src/app/account/account-reports/account-report-setup/account-report-setup.component.ts

```diff
@@ -25,6 +26,8 @@ export class AccountReportSetupComponent {
   months: Array<Month> = Months;
   //TODO: Report years validation. Start < End (issue-1194)
```

Comment:
While we are at it, can you remove this old comment. This issue takes care of the Todo.